### ISSUE TITLE
Android app

### ISSUE THREAD

**Issue by steinbro** [openscape-community/openscape#30]
This is probably the most -requested feature, and perhaps the most intimidating to tackle. As an initial step, we need someone with enough knowledge/interest to outline exactly what major sub-steps are required to make this happen (replacing specific dependencies, translating components, etc.)

**Comment by brunoprietog** [comment]
If you eventually decide to do this, it would be interesting to consider React Native with Expo. This would allow development without Android Studio, and if the same feature parity is achieved, it could be used to launch on iOS and have a single code base. Also, using Expo you don't need to have a Mac to compile. As a blind NVDA user on Windows, that's what keeps me from learning Swift and collaborating on the project. Unfortunately Voiceover is not enough for me and I have not been able to switch to Mac.

Expo has some interesting packages ready that we can easily use like gyroscope, location and more.

What do you think?

**Comment by steinbro** [comment]
That sounds promising, but I'd still be concerned about the core nonvisual parts of the app, like 3D audio and a text-to-speech engine flexible enough to render that audio in 3D space. I wonder if there are cross-platform AR/VR libraries that can do this these days?

**Comment by brunoprietog** [comment]
You're right, I failed to consider this point. I don't know if text to speech bridge allows to do that. At least in the documentation I do not see it.

<https://docs.expo.dev/versions/latest/sdk/speech/>

**Comment by mpaine-act** [comment]
Hello, I have been looking into developing a Flutter package with converting this project into flutter/dart widget and bloc cubits, along with using MapLibre GL and perhaps Zammo GPT chatbot, with the goal of having iOS, Android, Windows and Web supported (eventually).

I am starting to get attention with transit agencies so they can maintain their GTFS-Pathways datasets (beyond the obvious use for accessible crowdsourced pathfinding).  

Do the original Microsoft research team ever ping this group and assist?  I was going to use my contacts at Microsoft to approach it from the top-down.
